src/main/java/uk/gov/hmcts/reform/em/npa/redaction/PdfRedaction.java

```diff
@@ -80,29 +80,24 @@ private Rectangle createRectangle(
             RectangleDTO rectangle) {
 
         PDPage page = document.getPage(pageNumber);
-        PDRectangle pageSize = page.getMediaBox();
+        PDRectangle pageSize = page.getCropBox();
+
+        float x = pixelToPointConversion(rectangle.getX());
+        float y = pixelToPointConversion(rectangle.getY());
+        float width = pixelToPointConversion(rectangle.getWidth());
+        float height = pixelToPointConversion(rectangle.getHeight());
+
+        // Adjust for top-left origin
+        float adjustedY = pageSize.getUpperRightY() - y - height;
+
         // when there is a rotation coordinate systems completely changes and they way rectangle is drawn changes
         if (page.getRotation() != 0) {
-            var calculatedX = pixelToPointConversion(rectangle.getX());
-            var calculatedY = (pageSize.getWidth() - pixelToPointConversion(rectangle.getY()))
-                    - pixelToPointConversion(rectangle.getHeight());
-            return new Rectangle(
-                    calculatedY,
-                    pageSize.getHeight() - calculatedX - pixelToPointConversion(rectangle.getWidth()),
-                    pixelToPointConversion(rectangle.getHeight()),
-                    pixelToPointConversion(rectangle.getWidth()));
+            var calculatedX = pageSize.getHeight() - x - width;
+            var calculatedY = pageSize.getWidth() - y - height;
+            return new Rectangle(calculatedY, calculatedX, height, width);
         }
 
-        pageSize.setLowerLeftX(page.getCropBox().getLowerLeftX() / 0.75f);
-        pageSize.setLowerLeftY(page.getCropBox().getLowerLeftY() / 0.75f);
-        pageSize.setUpperRightX(page.getCropBox().getUpperRightX() / 0.75f);
-        pageSize.setUpperRightY(page.getCropBox().getUpperRightY());
-        return new Rectangle(
-                pixelToPointConversion(pageSize.getLowerLeftX() + rectangle.getX()),
-                (pageSize.getHeight() - pixelToPointConversion(rectangle.getY()))
```

Comment:
If a smaller change is desired (rather than in this PR currently where I've tried to simplify things overall), just swapping `pageSize.getHeight()` to `getUpperRightY()` here also resolves the known reproducible issue.